spec/fixtures/bulletin_post.xml

```diff
@@ -0,0 +1,40 @@
+<bulletin>
```

Comment:
These don't feel like fixtures, since they describe the outcome of a test, rather than setting it up.

I think it would read more clearly if you built the expectation in the test (as before) and just substituted in the footer from `RequestBuilder.default_footer`.

like

```ruby
expect(req.body).to be_equivalent_to(
  %{
    <bulletin>
    <!-- some xml -->
    <footer><![CDATA[#{GovDelivery::RequestBuilder.default_footer}
]]></footer>
    </bulletin>
  }
```